backends/platform/sdl/opendingux/opendingux.cpp

```diff
@@ -91,6 +71,59 @@ static const Common::AxisTableEntry odJoystickAxes[] = {
 	{ nullptr,	       0,				    Common::kAxisTypeFull, nullptr	       }
 };
 
+class OpenDinguxGraphicsManager : public SurfaceSdlGraphicsManager {
```

Comment:
Could you move this into a separate file in backends/graphics/opendingux for consistency with the other ports?

Reply:
Done